docs/basics/object-spawning.md

```diff
@@ -53,6 +53,43 @@ When you set the destroyWithScene property to `false` it will be treated the sam
 
 [Learn more about Netcode Scene Management here](scenemanagement/scene-management-overview.md)
 
+:::caution You might find it useful to add a `GameObject` property in a `NetworkBehaviour` derived component that you use to assign a network prefab instance for dynamically spawning. You need to make sure to instantiate a new instance **prior** to spawning. If you attempt to just spawn the actual network prefab instance it can result in unexpected results.
```

Comment:
```suggestion
:::caution You might find it useful to add a `GameObject` property in a `NetworkBehaviour`-derived component to use when assigning a network prefab instance for dynamically spawning. You need to make sure to instantiate a new instance **prior** to spawning. If you attempt to just spawn the actual network prefab instance it can result in unexpected results.
```